grails-app/controllers/org/pih/warehouse/api/ApiController.groovy

```diff
@@ -116,6 +117,7 @@ class ApiController {
         User user = User.get(session?.user?.id)
         Location location = Location.get(session.warehouse?.id)
         String highestRole = user.getHighestRole(location)
+        def allLocations = locationService.getAllLocations()
```

Comment:
Do we really want to have it here? And should it be all locations? With this locally getAppContext takes 2s to process while without this it took for me less than 200ms. On obdev getAppContext right now takes a bit less than 600 ms (from which major part was just initial connection). I don't like the fact that getAppContext is overloaded with so much info right now. I think it should be done in a separate endpoint. In fact, there already is `/api/locations`.